compiler/compiler_test.go

```diff
@@ -93,6 +101,7 @@ func TestCompiler(t *testing.T) {
 				Scheduler:          config.Scheduler(),
 				FuncImplementation: config.FuncImplementation(),
 				AutomaticStackSize: config.AutomaticStackSize(),
+				DefaultStackSize:   4096,
```

Comment:
I think this should match the target JSON for consistency.

```suggestion
				DefaultStackSize:   config.DefaultStackSize(),
```